algorithms/MDLMDec/loader_.py

```
@@ -0,0 +1,338 @@
+                (len(example['tokenized_sequence']) >= pep_length[0]) &
+                (len(example['tokenized_sequence']) <= pep_length[1]) 
+                if 'tokenized_sequence' in example else True
+            )
+        
+        self.dataset = dataset
+
+        ###############
+        # Dataloaders #
+        ###############
+        num_workers = min(self.dataset['train'].n_shards, num_workers)
+        eval_collate_function = lambda x: collate_fn(x, custom_columns=custom_columns)
```

Ruff (0.16.1)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `eval_collate_function` as a `def`

(E731)